doc/package-notes.xml

```diff
@@ -704,6 +706,57 @@ overrides = super: self: rec {
 }; }
 </programlisting>
   </para>
+  <para>
+    WeeChat allows to set defaults on startup using the <literal>--run-command</literal>.
+    The <literal>configure</literal> method can be used to pass commands to the program:
+<programlisting>weechat.override {
+  configure = { availablePlugins, ... }: {
+    init = ''
+      /set foo bar
+      /server add freenode chat.freenode.org
+    '';
+  };
+}</programlisting>
+    Further values can be added to the list of commands when running
+    <literal>weechat --run-command "your-commands"</literal>.
+  </para>
+  <para>
+    Additionally it's possible to specify scripts to be loaded when starting <literal>weechat</literal>.
+    These will be loaded before the commands from <literal>init</literal>:
+<programlisting>weechat.override {
+  configure = { availablePlugins, ... }: {
+    scripts = with pkgs.weechatScripts; [
+      weechat-xmpp weechat-matrix-bridge wee-slack
+    ];
+    init = ''
+      /set plugins.var.python.jabber.key "val"
+    '':
+  };
+}</programlisting>
+    Please keep in mind that <literal>scripts</literal> doesn't require store paths.
+    WeeChat simply needs to be able to find the specified scripts.
+  </para>
+  <para>
+    In <literal>nixpkgs</literal> there's a subpackage which contains derivations for
+    WeeChat scripts. Such derivations expect a <literal>passthru.scripts</literal> attribute
+    which contains a list of all scripts inside the store path. Furthermore all scripts
+    have to live in <literal>$out/share</literal>. An exemplary derivation looks like this:
+<programlisting>{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "exemplary-weechat-script";
+  src = fetchurl {
+    url = "https://scripts.tld/your-scripts.tar.gz";
+    sha256 = "...";
+  };
+  passthru.scripts = [ "foo.py" "bar.lua" ];
+  installPhase = ''
+    mkdir $out/share
```

Comment:
I would prefer those variables quoted, although there was some contoversy about that. So I won't insist on it.
Also applies to other files changed here.

Reply:
Admittedly I've never seen this pattern. Furthermore it's IIRC documented in Nixpkgs (or even Nix?), that `$out` exists as reference to the pending store path.

As you mention "some controversy about that", would you mind sending me the link to the discussion? I don't think that this is needed, but I'd like to see the arguments of both sides :)

Reply:
Yes, `$out` is a reference to the pending store path. I think as a shell variable it should still be quoted. I was referring to this discussion: https://github.com/NixOS/nixpkgs/pull/45657#discussion_r212835434